### PR TITLE
Yuvraj 3

### PR DESCRIPTION
**Pull request by 07yuvraj**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@
 ## Stack based problems
 
 - [Largest rectangular area under histogram](stack/Largest_rect_area_under_histogram.cpp)
+-[Infix to Postfix Conversiion](stack/Infix_to_Postfix.c)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md:123]: Typo in the spelling of conversion

File: C-Plus-Plus/stack/Infix_to_Postfix.c [hunk 2]
```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/Infix_to_Postfix.c:1]: Add a description about what your algorithm does

File: C-Plus-Plus/stack/Infix_to_Postfix.c [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    t=(struct Node*)malloc(sizeof(struct Node));
+
+    if(t==NULL)
+        printf("stack is full\n");
+    else{
+        t->data=x;
+        t->next=top;
+        top=t;
+    }
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/Infix_to_Postfix.c:23]: Format the Code using a C++ Formatter

File: C-Plus-Plus/stack/Infix_to_Postfix.c [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        else{
+            if(pre(infix[i])>pre(top->data))
+                push(infix[i++]);
+            else
+                postfix[j++]=pop();
+        }
+    }
+    while(top!=NULL)
+        postfix[j++]=pop();
+    postfix[j]='\0';
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/Infix_to_Postfix.c:90]: Put apt comments here

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
Are you working on the same @07yuvraj ?

**Comment by 07yuvraj** [comment]
> Are you working on the same @07yuvraj ?

Actually this was done by mistake I changed the readme file and it was done on another branch

**Pull request closed without merging**